resources/geocoding/en/358.txt

```diff
@@ -13,8 +13,9 @@
 # limitations under the License.
 
 # Generated from:
+# http://www.viestintavirasto.fi/attachments/maaraykset/Maarays_32.pdf [2014-11-23]
 # http://en.wikipedia.org/wiki/+358 [437010438]
-# Expanded 3585 prefix to exclude mobile.
+# Some prefixes expanded to exclude mobile/service/national numbers.
```

Comment:
Change "mobile/service/national" to "mobile/nationwide" to minimize confusion between this and document.